accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -92,72 +102,101 @@ static List<Packet> generate(RandomSource random, Function<? super CommandStore,
 
         for (int count = 0 ; count < operations ; ++count)
         {
+            int finalCount = count;
             Id client = clients.get(random.nextInt(clients.size()));
             Id node = nodes.get(random.nextInt(nodes.size()));
 
             boolean isRangeQuery = random.nextBoolean();
+            String description;
+            Function<Node, Txn> gen;
```

Comment:
since we don't know what topologies exist until we run on a node, moved the current logic into a lambda that runs in the node so we know what prefixes could be accessed...